truss/cli/cli.py

```diff
@@ -417,6 +434,44 @@ def deploy(
         )
 
 
+@chains.command(name="init")  # type: ignore
+@click.option(
+    "--name",
```

Comment:
given that `name` here doesn't really do anything (i.e. it is not the name of the chain - that is defined in the code itself), what are our thoughts on renaming this "directory" or "target_directory"?

Reply:
I think `--target-directory` is much clearer than `name`